src/main/java/step02/InputNumberRule.java

```diff
@@ -0,0 +1,31 @@
+package step02;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static step02.LottoConfig.*;
+
+public class InputNumberRule extends NumberRule {
+    List<Integer> inputNumber;
+
+    public InputNumberRule(String inputNumber) {
+        this.inputNumber = splitNumber(inputNumber);
+    }
+
+    @Override
+    public List<Integer> makeNumbers() {
+        return inputNumber;
+    }
+
+    private List<Integer> splitNumber(String numbers) {
+        List<Integer> lottoNumbers = new ArrayList<>();
+        String[] stringNumbers = numbers
+                .replace(BLANK, NO_BLANK)
+                .split(REGEX);
+        for (String number : stringNumbers) {
+            lottoNumbers.add(Integer.parseInt(number));
+        }
+
+        return lottoNumbers;
```

Comment:
람다식을 이용하여 한번에 변경 가능할 것 같습니다 😄 
이 부분은 참고만 부탁 드립니다 🙇 
```suggestion
        return Arrays.stream(numbers.trim().split(REGEX))
                .map(Integer::parseInt)
                .collect(Collectors.toList());
```